tests/BenchmarkDotNet.Tests/Portability/Cpu/CpuInfoFormatterTests.cs

```diff
@@ -0,0 +1,39 @@
+﻿#if CLASSIC || NETCOREAPP2_0
+using System.Text;
+using ApprovalTests;
+using ApprovalTests.Namers;
+using ApprovalTests.Reporters;
+using BenchmarkDotNet.Portability.Cpu;
+using BenchmarkDotNet.Tests.Mocks;
+using Xunit;
+
+namespace BenchmarkDotNet.Tests.Portability.Cpu
+{
+    [UseReporter(typeof(XUnit2Reporter))]
+    [UseApprovalSubdirectory("ApprovedFiles")]
+    public class CpuInfoFormatterTests
```

Comment:
If you have more than one approval test class, you should mark all of them with `[Collection("ApprovalTests")]`, it prevents running them in parallel. We have a problem with `NamerFactory.AdditionalInformation`: it's a static singleton. If you run each test individually, everything is OK. In the case of multithreading execution, the `AdditionalInformation` value from one test can be used in another.
See also: https://xunit.github.io/docs/running-tests-in-parallel.html

Reply:
FIxed